charger/ocpp.go

```diff
@@ -417,23 +418,15 @@ func (c *OCPP) currents() (float64, float64, float64, error) {
 }
 
 // Phases1p3p implements the api.PhaseSwitcher interface
-func (c *OCPP) Phases1p3p(phases int) error {
-	if !c.phaseSwitching {
-		c.log.DEBUG.Printf("phase switching during transaction is not supported by the charger")
-	}
-
-	c.log.TRACE.Printf("switching phase to %dp", phases)
-	c.phases = phases
-
+func (c *OCPP) phases1p3p(phases int) error {
 	enabled, err := c.Enabled()
 	if err == nil && enabled {
+		c.phases = phases
+
+		// TODO remove log
 		c.log.TRACE.Printf("sending request to set current %f @ %dp", c.current, c.phases)
-		if c.phaseSwitching {
-			err = c.setPeriod(c.current, c.phases)
-		} else {
-			c.Enable(false)
-			c.Enable(true)
-		}
```

Comment:
This enables phase switching for chargers that doesn't have capability to do it during transaction. I think it is worth keeping.

Reply:
The function will never be called for these chargers.